change/@fluentui-react-popover-7c1b5c26-eb72-4d8b-8951-f6a5501eb7af.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "fix: Update usages of react-positioning",
```

Comment:
I guess the comment should state `Breaking Change` to have this in our changelog ?

Reply:
yeah you're right, the current breaking changes will make it through components. I'll update once we have a full idea of if we want to accept all the breaks in the offline meeting

Reply:
```suggestion
  "comment": "BREAKING: Updates related to internal bump of Popper.js to Floating UI",
```